web_app/calendar/app.py

```diff
@@ -43,6 +46,46 @@ def set_form():
     return redirect(url_for_redirect)
 
 
+# Delete task
+@app.route(tasks_delete_route, methods=['get'])
+def tasks_deleter(task_id):
```

Comment:
Normally, English language does not have such word as 'deleter'. Rename it to 'task_remover' or something like that.